File: src/main/java/tech/zolhungaj/amqapi/servercommands/gameroom/lobby/PlayerLeftQueue.java
package tech.zolhungaj.amqapi.servercommands.gameroom.lobby;

import com.squareup.moshi.Json;
import tech.zolhungaj.amqapi.servercommands.CommandType;

@CommandType("player left queue")
public record PlayerLeftQueue(
        @Json(name = "name") String playerName
){}

File: src/main/java/tech/zolhungaj/amqapi/servercommands/gameroom/lobby/PlayerLeftQueue.kt
package tech.zolhungaj.amqapi.servercommands.gameroom.lobby

import com.squareup.moshi.Json
import tech.zolhungaj.amqapi.servercommands.CommandType

@JvmRecord
@CommandType("player left queue")
data class PlayerLeftQueue(
    @Json(name = "name")
    val playerName: String
)
